package.json

```diff
@@ -89,12 +88,12 @@
         "@babel/plugin-syntax-object-rest-spread": "^7.2.0",
         "@babel/preset-env": "^7.5.5",
         "@babel/register": "^7.5.5",
-        "@datadog/datadog-ci": "^0.7.1",
+        "@datadog/datadog-ci": "0.11.7",
```

Comment:
any reason to pin these to specific versions instead of using compatible with ^version?

Reply:
hey @nsollecito, there was no reason to pin those specific versions.   upgrading these package with yarn must have removed the caret.  this is updated now, thanks!